systems/messaging/node-feeder/pkg/listener.go

```diff
@@ -212,10 +208,9 @@ func (q *QueueListener) isRetryLimitReached(delivery amqp.Delivery) bool {
 	return false
 }
 
-// return error only if it could be fixed by retry
-// malformed request should not be considered as error
+//process nodeUpdateRequest msg also
 func (q *QueueListener) processRequest(delivery amqp.Delivery) error {
-	request := &DevicesUpdateRequest{}
+	request := &NodeUpdateRequest{}
 	err := json.Unmarshal(delivery.Body, request)
```

Comment:
Again it's protobuffer you receive here of type any.  Look at line 144 https://github.com/ukama/ukama/blob/main/systems/services/msgClient/internal/queue/listener.go
```
e := &pb.Event{
		RoutingKey: d.RoutingKey,
		Msg:        evtAny,
	}
```

evtAny is your NodeFeederMsg:

```
Something like this :
```
	msg := &pb.NodeFeederMessage{
				Target:     c.OrgName + "." + meteData.network + "." + meteData.site + "." + n,
				HTTPMethod: "POST",
				Path:       "/v1/configd/config",
				Msg:        anyMsg,
			}
```
Again you can have a loog at configurator configstore package to see how the protbuffer are packed and you have to follow reverse process here to unpack.

Reply:
Fixed